qa/L0_shared_memory/shared_memory_test.py

```diff
@@ -323,6 +324,29 @@ def test_unregisterall(self):
             self.assertTrue(len(status_after.regions) == 0)
         self._cleanup_server(shm_handles)
 
+    def test_infer_offset_out_of_bound(self):
+        # Shared memory offset outside output region - Throws error
+        error_msg = []
+        shm_handles = self._configure_sever()
+        if _protocol == "http":
+            # -0x20 in int64_t
+            offset = 18446744073709551584
+        else:
+            # grpc will throw if > int64, instead test for exceed shm region
```

Comment:
int64 or 64?

Reply:
The gRPC type is int64, so if that is overflowed (if > 2^63 - 1), then gRPC will throw an error, so I try to test for a off-by-1 corner instead on the gRPC connection.

I can change the wording if it is confusing.